libvmaf/src/feature/cambi.c

```diff
@@ -132,6 +136,16 @@ static const VmafOption options[] = {
         .min = 0.0001,
         .max = 1.0,
     },
+    {
+        .name = "max_log_contrast",
+        .help = "Max contrast luma level at 10-bits, set to  2^max_log_contrast \
+                 \t 0 to 5: Default 2 is recommended for banding from compression",
```

Comment:
You could do a multiline string like this: https://github.com/Netflix/vmaf/blob/master/libvmaf/src/feature/integer_adm.c#L52-L53

Reply:
got it